qiskit/pulse/__init__.py

```diff
@@ -130,15 +145,98 @@
 
 """
 
-from .channels import (DriveChannel, MeasureChannel, AcquireChannel,
-                       ControlChannel, RegisterSlot, MemorySlot)
-from .commands import AcquireInstruction, FrameChange, PersistentValue
-from .configuration import LoConfig, LoRange, Kernel, Discriminator
-from .exceptions import PulseError
-from .instruction_schedule_map import InstructionScheduleMap
-from .instructions import Acquire, Instruction, Delay, Play, ShiftPhase, Snapshot, SetFrequency
-from .interfaces import ScheduleComponent
-from .pulse_lib import (SamplePulse, Gaussian, GaussianSquare, Drag,
-                        Constant, ConstantPulse, ParametricPulse)
-from .pulse_lib.samplers.decorators import functional_pulse
-from .schedule import Schedule
+# Builder imports.
+from qiskit.pulse.builder import (
+    # builder construction methods.
+    active_backend,
+    active_transpiler_settings,
+    active_circuit_scheduler_settings,
+    append_instruction,
+    build,
+    num_qubits,
+    qubit_channels,
+    samples_to_time,
+    time_to_samples,
+    # builder instructions.
+    acquire,
+    barrier,
+    call,
+    call_schedule,
+    delay,
+    play,
+    set_frequency,
+    set_phase,
+    shift_frequency,
+    shift_phase,
+    snapshot,
+    # builder channels.
+    acquire_channel,
+    control_channels,
+    drive_channel,
+    measure_channel,
+    # builder contexts.
+    align_left,
+    align_right,
+    align_sequential,
```

Comment:
would it be possible to add `align_center` as well? I think that might be useful for example in different cross-resonances in parallel.

Reply:
I think I would like to do this at a later time. the semantics of what this means and how to implement it are unclear to me. Feel free to create a separate issue that describes how this would operate a bit better, eg., what happens when there are an even number of pulses, more then a single channel, etc.

Reply:
https://github.com/Qiskit/qiskit-terra/issues/4441